pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -977,4 +977,12 @@ self: super: {
   # https://github.com/bos/configurator/issues/22
   configurator = dontCheck super.configurator;
 
+  # Use global libsass library in order to use hsass in the repl
+  hlibsass = overrideCabal super.hlibsass (drv: {
+    configureFlags = drv.configureFlags or [] ++ ["-fexternalLibsass"];
+    extraLibraries = drv.extraLibraries or [] ++ [ pkgs.libsass ];
+  });
+
+  # Use global libsass library in order to use hsass in the repl
+  hsass = hsass.overrideScope (self: super: { hlibsass = self.hlibsass; });
```

Comment:
This line is unnecessary (and actually leads to an infinite loop). When `hlibsass` is overridden, then `hsass` binds to the modified version. That is the whole point of the override.